ui/components-react/src/components-react/filter-builder/Operators.ts

```diff
@@ -20,49 +19,53 @@ export enum PropertyFilterRuleGroupOperator {
 }
 
 /**
- * Operators for comparing property value in rule.
+ * Operators for comparing property value in [[PropertyFilterRule]].
  * @beta
  */
 export enum PropertyFilterRuleOperator {
```

Comment:
Any reason why this is an enum while `PropertyFilterBuilderRuleOperator` is a string literal union type? Should we stay consistent and use either enums + enum union OR literal union types?

Reply:
I want to use literal union but that breaks `presentation-components` as it uses `PropertyFilterRuleOperator` enum. I was hoping to switch `PropertyFilterRuleOperator` into literal union in next major AppUI release.

Reply:
Since you own the `presentation-components` could you just react to the change? Since we are making all these other breaking changes, i.e. some of the renames seem not to be necessary.

Reply:
I could change enum to literal union and react to these changes but `presentation-components` have peer dependency on `@itwin/components-react` specified as `^4.0.0` so removing enum in minor bump would mean that all previous `presentation-components` versions would break when used with new versions of `components-react`. Current approach keeps older versions working and allows to start using string literals instead of enum in new versions.

Reply:
We can keep it as is if you think it'll be too destructive, just a slight inconsistency :)

Just for argument's sake: this could be a good argument to NOT make these breaking changes to these beta APIs.
I.e. `PropertyFilterRuleGroupOperator` enum values were changed. What if your consumers used a literal number instead of an enum (even though it is unlikely)
```
function presentationFn(x: PropertyFilterRuleGroupOperator);
presentationFn(x.And); // continues to work
presentationFx(0); // OUCH after update
```

Reply:
Yes, that example would break but I have hope that raw numbers weren't used . It was my mistake to use enum from the start, so I am trying to fix that before promoting API to public. Removing enum in minor version proved to be too destructive so I hoped to move gradually: allow using enum or string literal and in major release leave only literal union.